lib/hasPermissions.ts

```diff
@@ -200,20 +206,51 @@ export class HasPermissionsDirectiveVisitor<
       location === DirectiveLocation.INPUT_OBJECT ||
       location === DirectiveLocation.ARGUMENT_DEFINITION;
 
+    const getDefaultValue = (
+      container: GraphQLArgument | GraphQLInputObjectType | GraphQLObjectType,
+      path?: Array<string>,
+    ): unknown => {
+      if ('defaultValue' in container) {
+        return container.defaultValue;
+      }
+      /**
+       * Used this, because the container is verified before and just can be Argument or InputObjectType
+       * and the path in a InputObjectType just can be undefined when the input doesn't have variables inside
+       * but this is don't accepted for sintax GraphQL, so is very difficult test this, therefore, the last
+       * case of the function is very difficult to test too, when the return is undefined;
+       */
+      /* istanbul ignore next */
+      if (isInputObjectType(container) && path && path.length > 0) {
+        const fieldName = path[path.length - 1];
+        const field = container.getFields()[fieldName];
+        return field.defaultValue;
+      }
+      /* istanbul ignore next */
+      return undefined;
```

Comment:
can't this be reached when used on `ARGUMENT_DEFINITION` ?
e.g.:
```graphql
query {
  myQuery(myArgument: String @hasPermissions(permissions:["nope"]))
}
```

Reply:
No, because this example is reached with this part of the code: 

`if ('defaultValue' in container) {
    return container.defaultValue;
 }`